src/onnx/onnx.cpp

```diff
@@ -765,27 +828,23 @@ struct onnx_parser
             op.lengths = std::vector<size_t>(in_lens.begin() + 2, in_lens.end());
         }
 
-        if(contains(info.attributes, "pads"))
+        // does not support ceil_mode
+        int ceil_mode = 0;
+        if(contains(info.attributes, "ceil_mode"))
         {
-            if(contains(info.attributes, "auto_pad"))
+            ceil_mode = info.attributes.at("ceil_mode").i();
```

Comment:
You can probably rewrite this so that you don't need to create an extra `ceil_mode` variable
```
if(info.attributes.at("ceil_mode").i() == 1) MIGRAPHX_THROW(...);
```

Reply:
fixed